scripts/fetch-icons.js

```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+import path from "path";
+import {exec} from "child_process";
+
+const iconsFolder = `./src/Icons`;
+
+const icons = fs
+  .readdirSync(path.resolve("./", iconsFolder))
+  .filter((file) => file.includes(".svg"));
+
+fs.writeFileSync("./src/Icons/index.tsx", "");
+
+const formatSvg = (svgContent) => {
+  const modifiedSvgContent = svgContent
+    .replace(/fill="[^"]*"/g, "fill={color}")
+    .replace(/stroke="[^"]*"/g, "")
+    .replace(/width="[^"]*"/g, "width={size}")
+    .replace(/height="[^"]*"/g, "height={size}");
```

Comment:
Maybe variabilize the regex to be clearer and what it does ?

Reply:
What do you find unclear here?

Reply:
the regex purpose

Reply:
To replace the entire attribut, from the first letter to the last coma, including the content of the attribut.
So if a width have a différent size we still catch it for instance.

Reply:
Varabilize the regex for more clarity